test/simulator/misc/fusion-combo.js

```diff
@@ -35,14 +34,13 @@ describe('Fusion Bolt + Fusion Flare', function () {
 			[{species: 'Dragonite', ability: 'Multiscale', moves: ['roost']}, {species: 'Lugia', ability: 'Multiscale', moves: ['roost']}],
 		]);
 
-		battle.commitDecisions();
+		battle.makeChoices();
 
 		let bpModifiers = new Map();
 		battle.onEvent('BasePower', battle.getFormat(), -100, function (bp, attacker, defender, move) {
 			bpModifiers.set(move.id, this.event.modifier);
 		});
-		battle.commitDecisions();
-		battle.p1.chooseMove('fusionflare', 2).chooseMove('instruct', -1).foe.chooseDefault();
+		battle.makeChoices('move fusionflare 2, move instruct -1');
```

Comment:
```suggestion
		battle.makeChoices('move fusionflare 2, move instruct -1', 'default');
```

Reply:
(This is generally clearer.)

Reply:
True. Should the `makeChoices()` calls be changed to `makeChoices('default', 'default')`?